bin/install

```diff
@@ -10,7 +10,7 @@ function exit_error() {
 # Check for dependencies
 missing_bins=""
 for bin in sed git; do
-    which $bin >/dev/null 2>&1 || missing_bins="$missing_bins $bin"
+    $bin --version >/dev/null 2>&1 || missing_bins="$missing_bins $bin"
```

Comment:
Is this a reliable way of checking?  I explicitly listed these as requirements in my example controller userenv:

https://github.com/perftool-incubator/crucible/blob/master/workshop/controller.json

Reply:
What is "perl-Thread-Queue" in the requirements?

Reply:
It's a thread safe queue library I typically use in my threaded perl programs.  It's used in bin/_logger.